### PR TITLE
Add XML libraries to prevent errors when converting JPEGs with EXIF data

### PR DESCRIPTION
**Pull request by stuartleylandcole**
Firstly thank you very much for providing these images, it has made local testing on my project far simpler than without it.

I encountered the following error when trying to convert some JPEGs:

```
convert: delegate library support not built-in '/images/sydney-opera-house.jpg' (XML) @ error/profile.c/ValidateXMPProfile/1927.
```

From some Googling this seems to be caused by the lack of libraries in the image to handle XML. I've therefore added `libxml2-dev` and `libxml2-utils` to the Debian images and `libxml2-devel` to the Fedora image. With these libraries included the image converts without any errors. I've included the file I used for testing and added it to the Travis script.

Please let me know if you need anything changing.

### CODE REVIEW

File: Dockerfile.buster [hunk 1]
```diff
@@ -12,7 +12,7 @@ RUN apt-get -y update && \
     # libheif
     libde265-0 libde265-dev libjpeg62-turbo libjpeg62-turbo-dev x265 libx265-dev libtool \
     # IM
-    libpng16-16 libpng-dev libjpeg62-turbo libjpeg62-turbo-dev libwebp6 libwebp-dev libgomp1 libwebpmux3 libwebpdemux2 ghostscript && \
+    libpng16-16 libpng-dev libjpeg62-turbo libjpeg62-turbo-dev libwebp6 libwebp-dev libgomp1 libwebpmux3 libwebpdemux2 ghostscript libxml2-dev libxml2-utils && \
```
Comment (dooman87) [inline Dockerfile.buster:15]: Thanks a lot for creating a PR! Do you reckon we could delete `libxml2-dev` once binary is built (line 37)?

**Review by dooman87 — COMMENTED** [review]
Thanks a lot for opening a PR! It looks great! Just have one small question and we can merge it after that

**Comment by stuartleylandcole** [comment]
@dooman87 good spot! I've removed `libxml2-dev` from both the Buster and Stretch images and verified that the image transforms without error.

Thanks for your quick review and apologies for taking some time to get back to you, I was away last week and didn't have my laptop with me.

**Comment by dooman87** [comment]
Thanks a lot for the contribution @stuartleylandcole! LGTM!

**Pull request merged**